R/radqlqc.R

```diff
@@ -571,21 +561,29 @@ get_qs_data <- function(ADSL,
   return(final_QS)
 }
 
-#' Function for generating random dates between 2 dates
+#' @describeIn h_adqlqc Function for generating random dates between 2 dates
 #'
-#' @param from start date/time variable
-#' @param to end date/time variable
-#' @param visit_id visit number variable
+#' @param from (`datetime vector`)\cr Start date/times.
+#' @param to (`datetime vector`)\cr End date/times.
+#' @param visit_id (`vector`)\cr Visit identifiers.
 #'
-#' @return data frame with new variable with random generated dates
-#'
-#' @importFrom lubridate as_datetime days
+#' @return Data frame with new randomly generated dates variable.
+#' @keywords internal
 #'
 #' @examples
-#' \dontrun{
-#' df <- group_by(df, USUBJID) %>%
-#'   mutate(ADTM = get_random_dates_between(TRTSDTM, TRTEDTM, AVISITN))
-#' }
+#' df <- dplyr::left_join(
+#'   ADSL,
+#'   QS,
+#'   by = c("STUDYID", "USUBJID"),
+#'   multiple = "all"
+#' ) |>
```

Comment:
are we committing to this new R piping instead of %>% ?

Reply:
but in the namespace it is imported right (`importFrom(magrittr,"%>%")`)? Does it break if you use it here?

Reply:
Yeah, I'm not sure why but it won't let me use it in examples (and I think this is the only example that would use it)